rs/config/src/execution_environment.rs

```diff
@@ -132,6 +132,9 @@ pub const MAX_COMPILATION_CACHE_SIZE: NumBytes = NumBytes::new(10 * GIB);
 /// Maximum number of controllers allowed in a request (specified in the interface spec).
 pub const MAX_ALLOWED_CONTROLLERS_COUNT: usize = 10;
 
+/// Maximum number of canister snapshots that can be stored for a single canister.
+pub const MAX_NUMBER_OF_CANISTER_SNAPSHOTS: usize = 1;
```

Comment:
I feel the name leaves it up to interpretation if it's a limit per canister or globally. How about `MAX_NUMBER_OF_SNAPSHOTS_PER_CANISTER`?

Reply:
Agreed, done.